1-exercises/A-reading-list/script.js

```diff
@@ -1,4 +1,23 @@
 function readingList(books) {
+  let content = document.querySelector("#content");
```

Comment:
Hi Petar, you did it great! You also can use `.getElementById("content");`

Reply:
Thank you :)